Formula/aerc.rb

```diff
@@ -4,6 +4,7 @@ class Aerc < Formula
   url "https://git.sr.ht/~rjarry/aerc/archive/0.11.0.tar.gz"
   sha256 "3d8f3a2800946fce070e3eb02122e77c427a61c670a06337539b3e7f09e57861"
   license "MIT"
+  revision 1
```

Comment:
Why do we need a `revision` bump? Are existing builds broken?

Reply:
Given the explanation below: it looks like new bottles are necessary to ship more stuff that was not enabled by the older `make` version.